style.css

```diff
@@ -640,6 +710,35 @@ section ul .linked-in,
     background: var(--color-white);
   }
 
+  /* -- -------------------Styling the Popup Window--------------------- -- */
+  .overlay {
+    max-width: 1440px;
+    max-height: 1280px;
+    position: absolute;
+    top: 0;
+    padding: 0;
+    background-color: rgba(193, 199, 208, 0.884);
+  }
+
+  .pop {
+    max-width: 1156px;
+    max-height: 1163px;
+    background-color: #fff;
+    margin: 77px 142px 337px 142px;
+    padding: 14px;
+    border-radius: 8px;
+  }
+
+  .details {
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+    gap: 24px;
+  }
+
+  .previewBtns {
+    justify-content: space-between;
+  }
+
```

Comment:
- [ ] You are almost there, @Amen-Musingarimi :muscle: . Your mobile pop-up is looking good now, you just have to see how to implement some of the suggestion of the previous reviewer as it has to do with your design and the project requirement. The pop-up is not too good in bigger screens now. I think you should be careful with the use of `max-height/width` because of your content does not grow as much as you needed (project requirement), it will not grow by itself. Please consider playing with `min-width/height` to ensure it conforms with the design. Also to match design, I think you should check the padding on the pop-up too. Great work :+1: Here is a figma design with some annotations that can guide you on necessary adjustments:

![image](https://user-images.githubusercontent.com/99715640/216386554-f826903c-bdc1-4100-874d-39f03f750f4b.png)

Reply:
And here, the use of min-height will help to stretch the elements to desired length before adjusting to fit content.